README.md

```diff
@@ -193,7 +193,7 @@ Show the code rustc generates for any function
 
 
 
-You can start by running `cargo asm` with no parameters - it will suggests how to narrow the
```

Comment:
those changes should go into `README.tpl`

Reply:
thx!